scripts/resinhup/run-resinhup-ssh.sh

```diff
@@ -46,11 +46,11 @@ Options:
         Run run-resinhup.sh with --remote . See run-resinhup.sh help for more details.
 
   --hostos-version <HOSTOS_VERSION>
-        Run run-resinhup.sh with --version .  See run-resinhup.sh help for more details.
+        Run run-resinhup.sh with --version . See run-resinhup.sh help for more details.
```

Comment:
Since you are here, you could also remove the extra space between the word "version" and the "." char.